public/js/controllers/coding_progress_table_controller.js

```diff
@@ -88,6 +104,61 @@ export class CodingProgressTableController {
                     parentNode.attr("class", "coding-complete");
                 }
             });
+
+            // Attach event listeners to columns' checkbox & ability to filter table
+            d3.selectAll("input[type=checkbox]").each(function(d, i, n) {
```

Comment:
checkbox labels should select the texts too

Reply:
updated thanks